CHANGELOG.MD

```diff
@@ -0,0 +1,43 @@
+# Change Log
+## 1.1 Release
+	- Supports Houdini 18.0 and USD 19.11
+	- Uses RPR SDK 1.34.5
+	- Uses RIF 1.4.3
```

Comment:
Let's move it to README.md

I was just thinking about it today. Why don't we update RPR and RIF version in README? There is a special column for that. It would be very convenient to track it in such a way so that when you are going through commits trying to find when exactly things get broken. Right now you have to lookup previous commits finding the commit that looks like "Update to RIF (version)" to get the version of SDKs that was used to build current commit. Of course, the submodule of RadeonProRenderThirdPartyComponents would do it much better, but until we can use it, let's use README to track the current version of dependencies.